pkgs/top-level/all-packages.nix

```diff
@@ -21862,6 +21862,8 @@ in
 
   imgcat = callPackage ../applications/graphics/imgcat { };
 
+  img-cat = callPackage ../applications/graphics/img-cat { };
```

Comment:
I don't like that they are named the same. This could cause confusion. Maybe someone else can say something about this?

Reply:
I guess you're right, this can cause some confusion, especially as both programs are basically doing the same thing: outputting images on the terminal. However, this implementation achieves a higher display resolution by combining fore- & background colors on half-height blocks and graphic characters.

Maybe renaming the nix-package to `imgcat-go` better points out that this is the specific implementation written in go. 

cc @trashhalo